### ISSUE TITLE
Full Front-End editing 

### ISSUE THREAD

**Issue by kiranetph** [Serhioromano/cobalt-9#109]
Since Cobalt 9 is about "Front-End" editing, shouldn't the "configuration" of Cobalt also be accessible from the Dashboard and be removed from the Back-end?

Maybe later in a C9.1 version, even the creation of C9 menu items through C9 menu item types should be possible through the control panel.

**Comment by Serhioromano** [comment]
I agree. Right now it is in a backend because it is automated. I only need to create config.xml in root and configuration works in Joomla.

Another reason is the central Joomla configuration point. All extensions are configured there in one place. So it is convenient. 

But alternative configuration form on the frontend would be nice. Especially right now we do not have to save the configuration at least once. It can immediately start working.

Fo menu links, yes I think, I can advance it. For example, in the article control menu I can place "Make a link to this article in menu". In sections manager, I can select a section and also link to menu and so on. 

And I consider this type of feature as a priority. Before I focused on logical features and ignored features like this. I sacrificed convenience and usability for new functionality. But this release I want to focus on usability and look & feel.

So I make it priority.

**Comment by kiranetph** [comment]
I believe it is the right choice .. 
I am researching different types of admin control panels to get some inspiration for a more "modern" approach to it. example : 
https://search.muz.li/inspiration/dashboard-inspiration/
I also got admin templates from theme forrest for demo and code.

A good start was already the conditional parameters (hiding not needed parameters) to make Cobalt less intimidating in reducing the visible number of parameters.

Another step would be, improving of default parameters of a default list template. Some of the info should not even be displayed as default like C/B/V/V/H. They create more confusion for first timers. 
![image](https://user-images.githubusercontent.com/15226809/64010105-b0c1fe00-cb4b-11e9-825a-10e34ecf8b8c.png)

I will open an extra issue on this.
